projects/vdk-plugins/vdk-ingest-http/src/vdk/plugin/ingest_http/ingest_http_plugin.py

```diff
@@ -8,16 +8,33 @@
 from vdk.api.plugin.hook_markers import hookimpl
 from vdk.internal.builtin_plugins.ingestion.ingester_base import IIngesterPlugin
 from vdk.internal.builtin_plugins.run.job_context import JobContext
+from vdk.internal.core.config import ConfigurationBuilder
 from vdk.plugin.ingest_http.ingest_over_http import IngestOverHttp
 
-
 log = logging.getLogger(__name__)
 
 
+@hookimpl
+def vdk_configure(config_builder: ConfigurationBuilder) -> None:
+    """
+    Here we define what configuration settings are needed for Greenplum with reasonable defaults
+    """
+    config_builder.add(
+        key="INGEST_OVER_HTTP_COMPRESSION_THRESHOLD_BYTES",
+        default_value=None,
+        description="Threshold used for determining if the payload will be compressed or not",
+    )
+    config_builder.add(
+        key="INGEST_OVER_HTTP_COMPRESSION_ENCODING",
+        default_value="utf-8",
+        description="Encoding used if compressing the payload",
```

Comment:
```suggestion
        description="Encoding used when compressing the payload",
```

Reply:
Let me elaborate the threshold description, so that it would be more clear that compression might be enabled/disabled. Then, "if" makes more sense I guess